packages/mui-joy/src/Grid/Grid.tsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import { createGrid } from '@mui/system/Grid';
+import { OverridableComponent } from '@mui/types';
+import { styled, useThemeProps } from '../styles';
+import { GridTypeMap } from './GridProps';
+
+const Grid = createGrid({
+  createStyledComponent: styled('div', {
+    name: 'MuiGrid',
+    overridesResolver: (props, styles) => styles.root,
+  }),
+  componentName: 'MuiGrid',
+  useThemeProps: (inProps) => useThemeProps({ props: inProps, name: 'MuiGrid' }),
+}) as OverridableComponent<GridTypeMap>;
+
+Grid.propTypes /* remove-proptypes */ = {
```

Comment:
Looks like some prop types are missing here, no?

Reply:
updated to include `*Offset` and `disableEqualOverflow`.